daemon/model-dbus-impl.cc

```diff
@@ -297,44 +284,49 @@ gdbus_cb_model_delete (MachinelearningServiceModel *obj,
 /**
  * @brief Event handler list of Model interface
  */
-static struct gdbus_signal_info handler_infos[] = {
-  {
-    .signal_name = DBUS_MODEL_I_HANDLER_REGISTER,
-    .cb = G_CALLBACK (gdbus_cb_model_register),
-    .cb_data = NULL,
-    .handler_id = 0,
-  }, {
-    .signal_name = DBUS_MODEL_I_HANDLER_UPDATE_DESCRIPTION,
-    .cb = G_CALLBACK (gdbus_cb_model_update_description),
-    .cb_data = NULL,
-    .handler_id = 0,
-  }, {
-    .signal_name = DBUS_MODEL_I_HANDLER_ACTIVATE,
-    .cb = G_CALLBACK (gdbus_cb_model_activate),
-    .cb_data = NULL,
-    .handler_id = 0,
-  }, {
-    .signal_name = DBUS_MODEL_I_HANDLER_GET,
-    .cb = G_CALLBACK (gdbus_cb_model_get),
-    .cb_data = NULL,
-    .handler_id = 0,
-  }, {
-    .signal_name = DBUS_MODEL_I_HANDLER_GET_ACTIVATED,
-    .cb = G_CALLBACK (gdbus_cb_model_get_activated),
-    .cb_data = NULL,
-    .handler_id = 0,
-  }, {
-    .signal_name = DBUS_MODEL_I_HANDLER_GET_ALL,
-    .cb = G_CALLBACK (gdbus_cb_model_get_all),
-    .cb_data = NULL,
-    .handler_id = 0,
-  }, {
-    .signal_name = DBUS_MODEL_I_HANDLER_DELETE,
-    .cb = G_CALLBACK (gdbus_cb_model_delete),
-    .cb_data = NULL,
-    .handler_id = 0,
-  }
-};
+static struct gdbus_signal_info handler_infos[]
```

Comment:
can we fix this line, like pipeline-dbus-impl.cc?
```
static struct gdbus_signal_info handler_infos[] = {
  {
      {
        ....
      },
  };
```

Reply:
Applied.